site/content/docs/user/known-issues.md

```diff
@@ -227,6 +228,12 @@ open /home/user/.docker/config.json: permission denied
 To fix this problem, either follow the docker's docs [manage docker as a non root user][manage docker as a non root user],
 or try to use `sudo` before your commands (if you get `command not found` please check [this comment about sudo with kind][sudo with kind]).
 
+
+## Docker init daemon config
+
+Please make sure that when you use `kind`, do not have `"init": true` in your `/etc/docker/daemon.json` because that will
```

Comment:
TIL you can even do this, I'm looking at the `docker run` docs and I don't see a way to opt back out of this when running a particular container (though maybe an explicit `--init=false`?). If it is possible to disable this per-container, then we should update kind to do this and avoid the issue altogether.

Reply:
I tested this and that seems to work!

- Enable `"init": true` in `/etc/docker/daemon.json`
- Run `docker run --rm -it --init alpine:3 ps aux ` -> should be pid 7 or something
- Run `docker run --rm -it --init=false alpine:3 ps aux ` -> should be pid 1

Reply:
![Screenshot_2021-02-04_16-55-33](https://user-images.githubusercontent.com/2277717/106918844-de7c8080-6709-11eb-990e-968f5575c8ce.png)

Reply:
@OhMyMndy I think that what Ben means is, that instead of fixing the documentation, you add the `--init=false` flag here https://github.com/kubernetes-sigs/kind/blob/e595a85bc454e88c5a0c12201bbeaab9fba51b19/pkg/cluster/internal/providers/docker/provision.go#L216-L241 
If the flag overrides the config, KIND will always create the containers the right way

Reply:
I understand, therefore I tried that solution and it is working for me.

Reply:
don´t you want to submit it to kind ?
this way, we don´t need the doc change

Reply:
> @OhMyMndy I think that what Ben means is, that instead of fixing the documentation, you add the --init=false flag here

Well, perhaps both. It would be nice to fix this so users don't have to change this setting in their docker settings, but it will take some time for the upgraded kind to filter out so we may still want to add a known-issues entry with the workaround.

We can also add this patch to kind ourselves, I just wanted to know if we can fix it without continuing to require you to change the docker settings (which seems to be so 👍 ).